src/sections/home/hero.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button, Flex, Text } from '@chakra-ui/react';
+import SNXMainLooperPlaceholder from 'src/svg/SNXMainLooperPlaceholder';
+
+export const Hero = () => {
+	let onClick = () => {};
+
+	if (typeof window !== 'undefined') {
```

Comment:
Slightly nicer to just put the `typeof window !== 'undefined'` inside onClick?

Reply:
+1